core/scss/utilities/mixins/grid/_grid-item-spans.scss

```diff
@@ -0,0 +1,47 @@
+@charset "UTF-8";
+
+//
+// @grid-item-spans($col-start, $col-end, $row-start, $row-end)
+//
+// Mixin to render cssGrid column spans in all modern browsers, including IE11.
+//
+// $col-start - integer: The column integer to start at
+// $col-end - list: A column integer or span
+// $row-start - integer: The row integer to start at
+// $row-end - list: A row integer or span
+//
+// Style guide: Mixins.Grid.grid-item-spans
+//
+@mixin grid-item-spans($col-start: null, $col-end: null, $row-start: null, $row-end: null) {
```

Comment:
I've adjusted this mixin to handle the difference between a `span  2` and a `2` end parameter and how ms handles that syntax.